src/main/features/response/guards/countyCourtJudgmentRequestedGuard.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+import User from 'idam/user'
+const logger = require('@hmcts/nodejs-logging').getLogger('response/guards/countyCourtJudgmentRequestedGuard')
+
+export default class CountyCourtJudgmentRequestedGuard {
+
+  static requestHandler (req: express.Request, res: express.Response, next: express.NextFunction) {
+    const user: User = res.locals.user
+    if (user.claim.countyCourtJudgmentRequestedAt) {
+      logger.warn('State guard: CCJ already requested - redirecting to dashboard')
+      return res.redirect(DashboardPaths.dashboardPage.uri)
```

Comment:
Need to check whether the dashboard is enough messaging, this isn't someone doing anything wrong its something that could easily happen.
Linking of a defendant to a claim is where I am most worried about this...

I would also suggest waiting till the next sprint so that we can finish everything in the current sprint...

Reply:
Agree, that is not perfect but protects us from inconsistency in case data. I would personally consider holding on with the release without this in place.